amp/parse.go

```diff
@@ -76,6 +91,20 @@ func parseInt(value string) int64 {
 	return 0
 }
 
+func parseBoolPtr(value string) *bool {
```

Comment:
Change to incorporate `strconv.ParseBool(x)` and return an error if there's a problem. We can accept empty string or a boolean value, anything else is an error.